README.md

```diff
@@ -99,7 +92,7 @@ Currently, network nodes are configured to rate limit high-volume publishing fro
 > **Important**  
 > If you are building a production-grade app, be sure to use an architecture that includes a local cache backed by an XMTP SDK.  
 
-To learn more, see [Use a local cache](https://xmtp.org/docs/tutorials/performance#use-a-local-cache).
+To learn more, see [Use local-first architecture](https://xmtp.org/docs/build/local-first).
```

Comment:
Has local-first architecture been built into the SDK - as with React, for example? Just curious!

Reply:
No. We haven't added infrastructure to allow the react native app to use the react hooks yet. The local storage work will likely come with V3 at this point.